ldcoolp/curation/inspection/readme/__init__.py

```diff
@@ -253,10 +258,12 @@ def retrieve_article_metadata(self):
         # Retrieve description (single string), strip vertical white space
         description = self.article_dict['item']['description'].replace('<div>', '')
         description = description.replace('</div>', '')
-        readme_dict['description'] = html2text(description)
-        # Strip extra white space from html2text
-        if readme_dict['description'][-2:] == "\n\n":
-            readme_dict['description'] = readme_dict['description'][:-2]
+        readme_dict['description'] = parser.handle(description)
+
+        # Clean whitespace around readme_dict keys
+        for k, v in readme_dict.items():
+            if isinstance(v, str):
+                readme_dict[k] = readme_dict[k].strip()
 
```

Comment:
I like this change, but unclear why this is done twice (here) and below.

Reply:
Ah, my bad. Just realized it was for two functions

Reply:
- Yes, I didn't think it was worth creating a method just for these two cases.
- I agree we need to have a specific set of deposits for edge cases.